README.md

```diff
@@ -107,6 +107,13 @@ If running as a job, we recommend using AWS credentials with minimum privileges
 }
 ```
 
+#### Check Account Exemptions
+**IAM Key Age**: Users may be exempted from the 90 day IAM key check test by including them in a comma-delimited list passed via the `CUTILS_IAM_KEY_WHITELIST`:
+```
+docker run -it --rm -e CUTILS_IAM_KEY_WHITELIST="service_acct1,service_acct2" cutils check_account
+```
+This is useful for users where long-lived IAM keys are by design or may be difficult to rotate.
```

Comment:
Perhaps change to something like: "Keys that are especially difficult to rotate can be whitelisted with this option.  Note, allowing keys to age past 90 days is not a recommended practice.  However, if it is absolutely necessary and instance roles are not an option (i.e., SES SMPT), we recommend that these keys be very tightly scoped in privilege."